pkg/noderesourcetopology/filter.go

```diff
@@ -223,14 +228,17 @@ func (tm *TopologyMatch) Filter(ctx context.Context, cycleState *framework.Cycle
 }
 
 func filterHandlerFromTopologyManager(conf nodeconfig.TopologyManager) filterFn {
-	if conf.Policy != kubeletconfig.SingleNumaNodeTopologyManagerPolicy {
-		return nil
+	if conf.Policy != kubeletconfig.SingleNumaNodeTopologyManagerPolicy &&
+		conf.Policy != kubeletconfig.BestEffortTopologyManagerPolicy &&
+		conf.Policy != kubeletconfig.RestrictedTopologyManagerPolicy {
+		return rejectNonSingleNUMANodeHandler
 	}
-	if conf.Scope == kubeletconfig.PodTopologyManagerScope {
+	switch conf.Scope {
+	case kubeletconfig.PodTopologyManagerScope:
 		return singleNUMAPodLevelHandler
-	}
-	if conf.Scope == kubeletconfig.ContainerTopologyManagerScope {
+	case kubeletconfig.ContainerTopologyManagerScope:
 		return singleNUMAContainerLevelHandler
+	default:
+		return nil
```

Comment:
if the scope is not explicitly set it should be container by default according to
https://kubernetes.io/docs/tasks/administer-cluster/topology-manager/#topology-manager-scopes